### PR TITLE
fix(curriculum): cat photo app workshop step 8 to 10

### LINKED ISSUE #56310: Update steps 8-10 of cat photo workshop to use specific assert methods where appropriate
We recently updated the guidelines for writing assertions here 
https://contribute.freecodecamp.org/how-to-work-on-coding-challenges/#writing-tests

For consistency and matching what is in the docs, we should slowly start to update the tests for the workshops and labs in the new frontend cert.

For this issue, just focus on updating the tests for steps 8-10 of the cat photo app workshop.
We don't want to have a large PR that updates a whole bunch of tests in a bunch of steps and makes it harder to review. 

Here is the link to the workshop:
https://github.com/freeCodeCamp/freeCodeCamp/tree/main/curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app

The tests should be updated, where appropriate, to use specific methods like `assert.equal`, `assert.exists`, `assert.match`, etc instead of just `assert`.

Here is the complete API reference 
https://www.chaijs.com/api/assert/#method_exists

Just focus on the new frontend cert and don't worry about updating the RWD cat photo app since that will become legacy anyway.

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56310

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690be902f051416f668962b.md [hunk 1]
```diff
@@ -24,31 +24,31 @@ Inside the existing `img` element, add a `src` attribute with this URL:
...
 ```js
-assert(document.querySelector('img').src === 'https://cdn.freecodecamp.org/curriculum/cat-photo-app/relaxing-cat.jpg');
+assert.equal(document.querySelector('img').src, 'https://cdn.freecodecamp.org/curriculum/cat-photo-app/relaxing-cat.jpg');
 ```
 
 Although you have set the `img` element's `src` to the correct URL, it is recommended to always surround the value of an attribute with quotation marks.
 
 ```js
-assert(!/\<img\s+src\s*=\s*https:\/\/cdn\.freecodecamp\.org\/curriculum\/cat-photo-app\/relaxing-cat\.jpg/.test(code));
+assert.isTrue(!/\<img\s+src\s*=\s*https:\/\/cdn\.freecodecamp\.org\/curriculum\/cat-photo-app\/relaxing-cat\.jpg/.test(code));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690be902f051416f668962b.md:45]: please use `assert.match` here

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690be902f051416f668962b.md [hunk 2]
```diff
@@ -24,31 +24,31 @@ Inside the existing `img` element, add a `src` attribute with this URL:
...
 ```
 
 Your `img` element should be closed either with `>` or `/>`.
 
 ```js
-assert(/<img\s+src\s*=\s*("|')https:\/\/cdn\.freecodecamp\.org\/curriculum\/cat-photo-app\/relaxing-cat\.jpg\1\s*\/?>/.test(code));
+assert.isTrue(/<img\s+src\s*=\s*("|')https:\/\/cdn\.freecodecamp\.org\/curriculum\/cat-photo-app\/relaxing-cat\.jpg\1\s*\/?>/.test(code));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690be902f051416f668962b.md:51]: `assert.match`

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690beec809b4f184cab6914.md [hunk 3]
```diff
@@ -37,28 +37,28 @@ Your anchor (`a`) element should be below the `p` element. You have them in the
...
+  document.querySelector('a').getAttribute('href'),
     'https://freecatphotoapp.com'
 );
 ```
 
 Although you have set the anchor ('a') element's `href` attribute to the correct link, it is recommended to always surround the value of an attribute with quotation marks.
 
 ```js
-assert(
+assert.isTrue(
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690beec809b4f184cab6914.md:61]: please use `assert.notMatch`

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
the title of the workshop is cat photo app, please change to that so that it is understandable for what workshop this issue is for

**Comment by gagan-bhullar-tech** [comment]
@ilenia-magoni updated PR

**Review by majestic-owl448 — APPROVED** [review]

**Review by huyenltnguyen — APPROVED** [review]

**Pull request merged**